runtime/jcl/common/mgmtthread.c

```diff
@@ -1344,6 +1359,17 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 
 	/* this may block on vm->managementDataLock */
 	getContentionStats(currentThread, targetThread, info);
+#if JAVA_SPEC_VERSION >= 19
+	/*
+	 * getStackFramePCs need to run on the continuation's stack to produce the right output.
+	 * To avoid potential issue on targetThread, this step is done just before getStackFramePCs
```

Comment:
```suggestion
	 * getStackFramePCs need to run on the carrier thread's stack to produce the right output.
	 * For threads with continuation mounted, copy the carrier thread's stack data from the
	 * continuation to a stack allocated J9VMThread structure. To avoid potential dependencies
	 * on targetThread pointer's address, the stack thread is only used for getStackFramePCs
```